### PR TITLE
Add event

### PR DESCRIPTION
**Pull request by shriharsha05**

### CODE REVIEW

File: src/pages/dash.vue [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <template>
-  <div class="announcements">
-    Announcements Page
+  <div class="dash">
```
Comment (mridulganga) [inline src/pages/dash.vue]: dash???
This should be notices

File: src/pages/addEvent.vue [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<template>
```
Comment (mridulganga) [inline src/pages/addEvent.vue:1]: This page should be linked from events.vue page,
dont make a separate nav item for this

File: src/App.vue [hunk 3]
```diff
@@ -28,24 +28,30 @@
...
+      <addEvent v-show="current_page=='addEvent'"></addEvent>
     </div>
   </div>
 </template>
 
 <script>
 import Events from "./pages/events.vue";
 import Members from "./pages/members.vue";
-import Announcements from "./pages/announcements.vue";
+import dash from "./pages/dash.vue";
```
Comment (mridulganga) [inline src/App.vue]: should be 
import Notices from "./pages/notices.vue"

File: src/App.vue [hunk 4]
```diff
@@ -28,24 +28,30 @@
...
+            @click="change_page('addEvent')"
+          >Add Event</a>
         </div>
       </div>
     </nav>
     <div class="main-container">
       <events v-show="current_page=='events'" :events_list="events" @reloadEvents="loadEvents"></events>
       <members v-show="current_page=='members'" :members_list="members" @reloadMembers="loadMembers"></members>
-      <announcements v-show="current_page=='announcements'"></announcements>
+      <dash v-show="current_page=='dash'"></dash>
```
Comment (mridulganga) [inline src/App.vue]: should be 
<notices current_page="NOTICES"></notices>

File: src/App.vue [hunk 5]
```diff
@@ -28,24 +28,30 @@
             @click="change_page('members')"
           >Members</a>
           <a
-            :class="{'navbar-item': true, 'is-active is-tab':current_page=='announcements'}"
-            @click="change_page('announcements')"
-          >Announcements</a>
+            :class="{'navbar-item': true, 'is-active is-tab':current_page=='dash'}"
+            @click="change_page('dash')"
```
Comment (mridulganga) [inline src/App.vue]: should be notices
The body of it can be announcements

File: src/App.vue [hunk 6]
```diff
@@ -28,24 +28,30 @@
...
+          >dash</a>
+          <a
+            :class="{'navbar-item': true, 'is-active is-tab':current_page=='addEvent'}"
+            @click="change_page('addEvent')"
+          >Add Event</a>
```
Comment (mridulganga) [inline src/App.vue]: No need for separate menu item for Add Events

**Review by mridulganga — CHANGES_REQUESTED** [review]

File: src/pages/addEvent.vue (hunk 2 above)
Comment (shriharsha05) [reply src/pages/addEvent.vue:1]: Help Needed

Comment (mridulganga) [reply src/pages/addEvent.vue:1]: I will implement routers 
That will make things very easy.
For now remove all changes except the addEvents file. When I make the changes, rebase and then work on that

**Pull request closed without merging**